### DISCUSSION TITLE
Status of openSUSE Leap support

### DISCUSSION THREAD

**Discussion by dermot-hardy** [prisma/prisma#24357] [Q&A] [answered]
### Question

Given #17947, is the current situation that Prisma supports [OpenSUSE Tumbleweed](https://www.opensuse.org/#Tumbleweed) but not [OpenSUSE Leap](https://www.opensuse.org/#Leap)?

**Comment by janpio** [comment, marked as answer]
There is no open issue for OpenSUSE Leap, so either it just works or no one has tried and reported that it does not.

**Comment by dermot-hardy** [comment]
Ok thank you.
